server/services/voip/service.ts

```diff
@@ -31,11 +32,25 @@ export class VoipService extends ServiceClassInternal implements IVoipService {
 
 		this.logger = new Logger('VoIPService');
 		this.commandHandler = new CommandHandler(db);
-		try {
-			Promise.await(this.commandHandler.initConnection(CommandType.AMI));
-		} catch (error) {
-			this.logger.error({ msg: `Error while initialising the connector. error = ${error}` });
-		}
+		this.init();
+	}
+
+	private async init(): Promise<void> {
+		settings.watch('VoIP_Enabled', (value) => {
+			try {
+				if (value) {
+					this.logger.info('Starting VoIP service');
+					Promise.await(this.commandHandler.initConnection(CommandType.AMI));
```

Comment:
You're in an async function, no need for `Promise.await` here 👀

Reply:
Done :)